### DISCUSSION TITLE
Moisture sensor

### DISCUSSION THREAD

**Discussion by pabinupc1981** [sebr/bhyve-home-assistant#175] [General]
Is there a way to integrate with moisture sensors?

**Comment by sebr** [comment]
Does b-hyve have a moisture sensor?

**Reply by TomAutoHome** [reply]
If you mean SENSOR terminal to connect wired or wireless rain or rain/freeze sensors - yes it has. But sensor activation should be reported as a rain delay. Mine on #57915 4-zone does not actually work (device ignores wet sensor input - open circuit does not trigger rain delay or turn off the valves when they work).
Or do you mean flood sensors? - That's a different story/device.
